guides/src/components/MarkdownPageFooter.js

```diff
@@ -0,0 +1,30 @@
+// --- Dependencies
+import * as React from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * Component
+ */
+
+export default function MarkdownPageFooter({ group, isIndex, section, title }) {
+  const baseUrl =
+    'https://github.com/spree/spree/blob/master/guides/src/content'
```

Comment:
```suggestion
    'https://github.com/spree/spree/edit/master/guides/src/content'
```

This way the user will land on the editor :)